code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -1,5 +1,5 @@
 /obj/item/radio/intercom
-	name = "station intercom"
+	name = "Station Intercom (General)"

```

Comment:
might ought to be `"\improper Station Intercom (General)"` or perhaps just take this opportunity to name station intercoms "station intercom" w/o the unnecessary capitalization

Reply:
yeah, it does look better